<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastro_endereco</class>
 <widget class="QMainWindow" name="Cadastro_endereco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Endereços</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>267</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="endereco_lista">
     <attribute name="title">
      <string>Lista</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_enderecos">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>48</y>
        <width>401</width>
        <height>191</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>9</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>-7</x>
        <y>10</y>
        <width>397</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>CEP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_pesquisar_cep">
         <property name="inputMask">
          <string>99999-999</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pesquisar">
         <property name="text">
          <string>Pesquisar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="endereco_cadastro">
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>38</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>161</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Cidade *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_cidade">
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>291</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Complemento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_complemento">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>211</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Rua *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_rua">
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>71</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Número *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_numero">
         <property name="inputMask">
          <string>99999</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>121</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Bairro *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_bairro">
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>74</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Pais *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_pais">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>CEP *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_cep">
         <property name="inputMask">
          <string>99999-999</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="maxLength">
          <number>9</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QRadioButton" name="radioButton_ativo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ativo</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_9">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>74</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Estado *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_estado"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_novo">
       <property name="text">
        <string>Novo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_gravar">
       <property name="text">
        <string>Gravar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_voltar">
       <property name="text">
        <string>Voltar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_pesquisar_cep</tabstop>
  <tabstop>pushButton_pesquisar</tabstop>
  <tabstop>tableWidget_enderecos</tabstop>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>radioButton_ativo</tabstop>
  <tabstop>lineEdit_cep</tabstop>
  <tabstop>lineEdit_rua</tabstop>
  <tabstop>lineEdit_numero</tabstop>
  <tabstop>lineEdit_bairro</tabstop>
  <tabstop>lineEdit_cidade</tabstop>
  <tabstop>comboBox_estado</tabstop>
  <tabstop>lineEdit_complemento</tabstop>
  <tabstop>comboBox_pais</tabstop>
  <tabstop>pushButton_novo</tabstop>
  <tabstop>pushButton_gravar</tabstop>
  <tabstop>pushButton_voltar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
